web/testing/chirality/chirality.js

```diff
@@ -61,9 +62,9 @@ function Keyboard_chirality() {
         if(keySymbol == "K_*") {
           continue;
         } else if(kls[layer][key] != '') {
-          if (k.KKM(e, modCode, k.osk.keyCodes[keySymbol])) {
+          if (k.KKM(e, modCode, osk.keyCodes[keySymbol])) {
```

Comment:
Does this become `kbdInterface.keyMatch`?

Reply:
From kmwcallback.js: 
```
var keymanweb=window['keyman'], kbdInterface=window['KeymanWeb']=keymanweb['interface'], KeymanWeb=kbdInterface;

//...

KeymanWeb['KKM'] = kbdInterface['keyMatch'] = kbdInterface.keyMatch = function(e,Lruleshift,Lrulekey) // ...
```

Technically speaking, both KKM and keyMatch are both fields of `k`.  Old keyboards will only know this as `k.KKM`, which must be supported through a legacy `KeymanWeb` variable, which is how `k` is set above.

The published API will instead point people to `keyman.interface.keyMatch`, which is a more user-friendly/documentation-friendly name for the same function.  `KeymanWeb === keyman.interface`.

If the initialization at the top is fun to parse, it goes like this.

```
var keymanweb=window['keyman'], kbdInterface=window['KeymanWeb']=keymanweb['interface'], KeymanWeb=kbdInterface;
```
*Left to right*
* `keymanweb` refers to the global variable `keyman`.  It used to refer to `tavultesoft.keymanweb`; reusing it with no name change means a lot less review clutter since we can leave many old lines in place as they were.
* Define a new global `KeymanWeb` (via `window`) and set it as an alias to `keyman.interface`.  Also set the local shorthand `kbdInterface` so that it's clear we're defining methods for use through `keyman.interface`.
* Create a direct alias for that local shorthand to clearly establish the method names provided for legacy support.

```
KeymanWeb['KKM'] = kbdInterface['keyMatch'] = kbdInterface.keyMatch = function(e,Lruleshift,Lrulekey) // ...
```
*Left to right*
* Define the legacy name for the keyboard interface function.
* Establish it as an exposed name for the API.  (b/c Google Closures is a thing)
* Establish it as an internal name for minification.  (b/c Google Closures is a thing)
* Define the function.

So, chirality.js is still using the legacy interface, same as all the other old keyboards.